src/irmin-pack/unix/pack_store.ml

```diff
@@ -92,9 +92,14 @@ struct
 
   let v ~config ~fm ~dict ~dispatcher =
     let indexing_strategy = Conf.indexing_strategy config in
-    let lru_size = Conf.lru_size config in
+    (* assume the default object size is the size of a hash *)
+    let obj_size = Hash.hash_size in
+    let lru_size = Conf.lru_size config * obj_size in
```

Comment:
Instead of making an assumption about default object size, I think we ought to add an explicit `lru_mem_size` configuration instead of trying to calculate it based on an entries configuration. `lru_size` would then continue to use a weight of 1 like today.